subsys/bluetooth/host/crypto.c

```diff
@@ -45,14 +45,19 @@ static int prng_reseed(struct tc_hmac_prng_struct *h)
 		rp = (void *)rsp->data;
 		memcpy(&seed[i * 8], rp->rand, 8);
 
+		explicit_bzero(rp->rand, 8);
 		net_buf_unref(rsp);
 	}
 
-	extra = k_uptime_get();
+	ret = cipher_prng_op(&prng_session, &(struct cipher_prng_pkt) {
+		.reseed = true,
+		.additional_input = seed,
+		.additional_input_len = sizeof(seed),
```

Comment:
it will not work with tinycrypt.  tc_hmac_prng_reseed() will require .data anda .data_len set.
additional_input should be mapped to extra and not to seed, the problem is that medtls is expecting seed be passed in additional_input. This must be changed too.